spec/index.bs

```diff
@@ -3444,33 +3458,46 @@ Issue: link to the semantics of SV_SampleIndex and SV_Coverage in WGSL spec.
-                        - |descriptor|.{{GPURenderPipelineDescriptor/alphaToCoverageEnabled}} is `false`.
-                </div>
+            1. Let |pipeline| be a new valid {{GPURenderPipeline}} object.
+            1. Issue the following steps on the [=Device timeline=] of |this|:
+                <div class=device-timeline>
+                    1. If any of the following conditions are unsatisfied:
+                        <div class=validusage>
+                            - |this| is a [=valid=] {{GPUDevice}}.
+                            - |descriptor|.{{GPUPipelineDescriptorBase/layout}} is [$valid to use with$] |this|.
+                            - [$validating GPUProgrammableStageDescriptor$]({{GPUShaderStage/VERTEX}},
+                                |descriptor|.{{GPURenderPipelineDescriptor/vertexStage}},
+                                |descriptor|.{{GPUPipelineDescriptorBase/layout}}) succeeds.
+                            - If |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}} is not `null`:
+                                - [$validating GPUProgrammableStageDescriptor$]({{GPUShaderStage/FRAGMENT}},
+                                    |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}},
+                                    |descriptor|.{{GPUPipelineDescriptorBase/layout}}) succeeds.
+                            - |descriptor|.{{GPURenderPipelineDescriptor/colorStates}}.length is less than
+                                or equal to 4.
+                            - [$validating GPUVertexStateDescriptor$](|descriptor|.{{GPURenderPipelineDescriptor/vertexState}},
+                                |descriptor|.{{GPURenderPipelineDescriptor/vertexStage}}) passes.
+                            - If |descriptor|.{{GPURenderPipelineDescriptor/alphaToCoverageEnabled}} is `true`:
+                                - |descriptor|.{{GPURenderPipelineDescriptor/sampleCount}} is greater than 1.
+                            - If the output SV_Coverage semantics is [=statically used=] by
+                                |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}}:
+                                - |descriptor|.{{GPURenderPipelineDescriptor/alphaToCoverageEnabled}} is `false`.
+                        </div>
 
-            Then:
-                1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
-                1. Create a new [=invalid=] {{GPURenderPipeline}} and return the result.
+                        Then:
+                            1. Generate a {{GPUValidationError}} in the current scope with appropriate
+                                error message.
+                            1. Make |pipeline| [=invalid=].
+
+                    1. Set |pipeline|.{{GPURenderPipeline/[[descriptor]]}} to |descriptor|.
+                    1. If |descriptor|.{{GPURenderPipelineDescriptor/primitiveTopology}} is
```

Comment:
I think we should have validation that `indexFormat` is set IFF the primitive topology is a strip.

Reply:
Done!